src/actions/querystringsearch/querystringsearch.js

```diff
@@ -21,6 +22,15 @@ export function getQueryStringResults(path, data, subrequest, page) {
     });
   }
 
+  // fixes https://github.com/plone/volto/issues/2397
+  if (requestData?.sort_order !== null) {
+    if (requestData.sort_order === true || requestData.sort_order === false) {
```

Comment:
```suggestion
    if (typeof requestData.sort_order === "boolean") {
```

How about this as a simpler check?